go/logic/ccql.go

```diff
@@ -38,8 +42,9 @@ func queryHost(host string, user string, password string, schema string, queries
 }
 
 // QueryHosts will issue concurrent queries on given list of hosts
-func QueryHosts(hosts []string, user string, password string,
-	defaultSchema string, schemas []string, queries []string,
+func QueryHosts(
+	hosts, schemas, queries []string,
+	user, password, defaultSchema string,
```

Comment:
I suspect the reason for reordering arguments was by their data types. I disagree on the reordering. Data types do not have semantics, I'd like ordering to return to what it used to be, consistent with `queryHost()`